.github/ISSUE_TEMPLATE/issue_template.md

```diff
@@ -0,0 +1,43 @@
+---
+name: Issue Template
+about: File an issue
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+<!--
+This is a suggested issue template for tedana.
+
+If there is other information that would be helpful to include, please don not hesitate to add it!
+
+Before submitting, please check to make sure that the issue is not already addressed.
```

Comment:
One question: should we ask folks to link to related issues ? That way we make sure that issues don't get too divergent, even if they're all around a central theme !

Reply:
Yes, I missed that in this revision. I'll put it back in now.

Reply:
Addressed in b0119239396a9191adf531ef7a56dea43424e19b